python/pyproject.toml

```diff
@@ -58,6 +57,7 @@ scikit-learn = {version = "^1.0.2", optional = true}
 # Fugue related dependencies
 fugue = {version = "^0.8.1", optional = true}
 requests = "^2.27"
+protobuf = "^4.22.0"
```

Comment:
This is still an unnecessary change to our install dependencies, we don't need to force users to upgrade their protobuf. 4.22 works but so do older versions of protobuf with the protoc version that builds the message definitions in our CI.